ui/src/app/components/toast-banner.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
```

Comment:
this file is mainly styling and display logic adapted from the previous version of status-alert-banner